app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -13,23 +13,21 @@ module AuthnJwt
 
       def call
         validate_and_decode_token
-        conjur_id
+        jwt_identity
```

Comment:
Inside the main `call`, the methods should be sentences and not nouns.
it should be read as a story to tell me what is going on.
But I can't understand what `jwt_identity` tells me; it's not a sentence, what does it mean?

Reply:
I wanted to make a get_jwt_identity function accross all the authenticator but in previous review was asked not to use get.

so i will rename to get only here